csrc/transformer/inference/csrc/pt_binding.cpp

```diff
@@ -452,14 +452,17 @@ std::vector<at::Tensor> ds_softmax_context(at::Tensor& query_key_value,
                                            unsigned layer_id,
                                            unsigned num_layers,
                                            at::Tensor& alibi,
-                                           float rope_theta)
+                                           float rope_theta,
+                                           bool is_prompt,
+                                           std::optional<at::Tensor> token_idx,
+                                           std::optional<at::Tensor> position_ids)
 {
     unsigned bsz = query_key_value.size(0);
     unsigned seq_len = query_key_value.size(1);
     int k = query_key_value.size(2) / (heads + 2 * (num_kv > 0 ? num_kv : heads));
     unsigned hidden_dim = heads * k;
 
-    bool is_prompt = (seq_len > 1);
+    is_prompt = (seq_len > 1);
```

Comment:
Why is the `is_prompt` argument ignored?

Reply:
The new parameters were added to align the signature of the cpp and python implementation. 
in python this argument is sent as a parameter for the function, while in the cpp implementation it can be deduced based on other variables. 
so instead of adding a non-used argument, I thought about using it is the same variable.

Reply:
Can you add this explanation to help future readers? Thanks!